streams.yml

```diff
@@ -61,37 +61,28 @@ rhel-7:
 # exclusively for testing are bundled on top of the golang builder image.
 rhel-7-ci-build-root:
   image: not_applicable
-  upstream_image_base: registry.ci.openshift.org/ocp/builder:rhel-7-golang-1.15-openshift-{MAJOR}.{MINOR}
+  upstream_image_base: registry.ci.openshift.org/ocp/builder:rhel-7-golang-1.16-openshift-{MAJOR}.{MINOR}
```

Comment:
Looks like we don't have 1.16 for rhel7 yet?

Reply:
correct we do not. yes, this will wreak a certain amount of havoc.

Reply:
uh... so i'll fix this line. i must have been confused. thanks!